pimpmycause/pimpmycause/templates/layout.html

```diff
@@ -55,10 +55,21 @@
                             <a href="https://twitter.com/pimpmycause"><i class="fa fa-twitter"></i></a>
                         </div>
 
-                        <!-- Sign in button -->
-                        <div class="header-sign-in">
-                            <a href="/accounts/login/?next=/" class="button round">Sign in</a>
-                        </div>
+                        {% if user.is_authenticated %}
+
+                            <!-- Sign in button -->
+                            <div class="header-sign-in">
+                                <a href="#" class="button"> {{ request.user.username }} </a>
```

Comment:
same here, If user is logged in then we display conditionally a username instead of sign in button.